src/core/reference/src/utils/phillox_converter.cpp

```diff
@@ -120,16 +120,16 @@ bfloat16 uint32_to_bfloat16(uint32_t x) {
 // uint32 value. Resulting value is in interval [0,1).
 template <typename T>
 T uint32_to_T_type_float(uint32_t x) {
-    auto MASK = static_cast<uint32_t>((static_cast<uint64_t>(1) << std::numeric_limits<float>::digits) - 1);
-    auto DIVISOR = static_cast<float>(1) / (static_cast<uint64_t>(1) << std::numeric_limits<float>::digits);
+    auto MASK = (1U << std::numeric_limits<float>::digits) - 1;
+    auto DIVISOR = static_cast<float>(1) / static_cast<float>(1U << std::numeric_limits<float>::digits);
```

Comment:
```suggestion
    constexpr auto masks = (1U << std::numeric_limits<float>::digits) - 1U;
    constexpr auto divisor = 1.0f / static_cast<float>(1U << std::numeric_limits<float>::digits);
```
Can be constexpr values. For variables `lawer_case` is used.

Reply:
Done!